root/static/scripts/edit/URLCleanup.js

```diff
@@ -765,6 +772,26 @@ const CLEANUPS = {
       url = url.replace(/^(?:https?:\/\/)?(?:www\.)?soundtrackcollector\.com\/.*\?composerid=([0-9]+).*$/, "http://soundtrackcollector.com/composer/$1/");
       return url;
     }
+  },
+  patronage: {
+    match: [
+      new RegExp("^(https?://)?(www\\.)?changetip\\.com/tipme/[^/?#]", "i"),
+      new RegExp("^(https?://)?[^/?#]+\\.tip.me([/?#].*)?$", "i"),
+      new RegExp("^(https?://)?(www\\.)?flattr\\.com/profile/[^/?#]", "i"),
+      new RegExp("^(https?://)?(www\\.)?patreon\\.com/[^/?#]", "i"),
+      new RegExp("^(https?://)?(www\\.)?paypal\\.me/[^/?#]", "i")
```

Comment:
Thank you, (no) trailing comma is mostly a style preference, good to know you accept both.